### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by tadash10**
#239   "Documentation update proposal" 

## Summary by CodeRabbit

- **Documentation**
  - Updated the contributing guide with new resources to aid understanding.
  - Included instructions for local documentation preview with `mkdocs serve`.
  - Added guidance on highlighting lines in code blocks and using admonitions for emphasis.

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -36,3 +36,18 @@ If you need ideas, you can check out the [help wanted](https://github.com/jxnl/i
 <a href="https://github.com/jxnl/instructor/graphs/contributors">
   <img src="https://contrib.rocks/image?repo=jxnl/instructor" />
 </a>
+
+
+## Additional Resources
```
Comment (jxnl) [inline docs/contributing.md:41]: can you mvoe this above the contributors table and something  like 

to run the docs pip install requirements-doc.txt and run `mkdocs serve -w .` to preview the documentation

to improve the legibility of code and content consider using 


hl_lines... ...
admonitinos: ...

and include a code example in our docs so we dont /have/ to jump links ( can literally copy paste a snippet of their markdown)

**Review by jxnl — CHANGES_REQUESTED** [review]
can you move this under `

File: docs/contributing.md (hunk 1 above)
Comment (tadash10) [reply docs/contributing.md:41]: Already make the requested changes and also leave in "Extended description
"  a second version of the corrections. Regarding (can literally copy paste a snippet of their markdown) if you can guide me how, I'll do it asap. thanks!

**Comment by tadash10** [comment]
A few days ago I updated according to the corrections and recommendations, any other changes or we are already satisfied?thanks and good week

**Comment by tadash10** [comment]
How's it going, I hope you have a good week. After Update contributing.md #241 merged, note that I am not yet listed as a contributor. If you want me to address something of your preference, let me know.

**Pull request merged**